libvips/convolution/im_aconv.c

```diff
@@ -1005,9 +1005,7 @@ aconv_horizontal( Boxes *boxes, IMAGE *in, IMAGE *out )
 
 #define CLIP_UCHAR( V ) \
 G_STMT_START { \
-	if( (V) < 0 ) \
-		(V) = 0; \
-	else if( (V) > UCHAR_MAX ) \
+	if( (V) > UCHAR_MAX ) \
```

Comment:
Ooop, I see you caught that one too. Is this change still necessary after your second patch?